teams/translatebot.md

```diff
@@ -0,0 +1,17 @@
+# Team: TranslateBot (dev2718, rafiv)
+
+Team: [translatebot](https://github.com/orgs/WoN-Hackathon-2019/teams/translatebot)
+
+Members:
+* https://github.com/dev2718
+* https://github.com/rafiv
+
+
+# Projects
+## Example Project
+We create a bot which will be used for translating messages between two people, e. g. Alice from Austria wants to communicate with Bob from Australia. Alice sends her messages in German, Bob in English and the bot translates them that Alice can read the messages from Bob in German and Bob the messages von Alice in English.
+
+Link: [translatebot](https://github.com/translatebot)
```

Comment:
is this the link to your actual github project for this course?

Reply:
no, sorry, i'll correct that